DescendantBooks/po/it-local.po

```diff
@@ -0,0 +1,739 @@
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1351
+msgid "Henry numbering"
+msgstr "Numerazione di Henry"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1352
+msgid "d'Aboville numbering"
+msgstr "Numerazione d'Aboville"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1354
+msgid "Record (Modified Register) numbering"
+msgstr "Numerazione a record (registro modificato)"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1364
+msgid "Page break between generations"
+msgstr "Interruzione di pagina tra due generazioni"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1366
+msgid "Whether to start a new page after each generation."
+msgstr "Indica se iniziare una nuova pagina dopo ogni generazione."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1369
+msgid "Page break before end notes"
+msgstr "Interruzione di pagina prima delle annotazioni"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1371
+msgid "Whether to start a new page before the end notes."
+msgstr "Indica se iniziare una nuova pagina prima delle annotazioni."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1378
+msgid "Content"
+msgstr "Contenuto"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1380
+msgid "Use callname for common name"
+msgstr "Usa il soprannome come nome comune"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1381
+msgid "Whether to use the call name as the first name."
+msgstr "Indica se usare il soprannome come primo nome."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1384
+msgid "Use full dates instead of only the year"
+msgstr "Usa date complete anziché solamente l'anno"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1386
+msgid "Whether to use full dates instead of just year."
+msgstr "Indica se usare date complete al posto del solo anno."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1389
+msgid "List children"
+msgstr "Lista dei figli"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1390
+msgid "Whether to list children."
+msgstr "Indica se listare i figli oppure no."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1393
+msgid "Compute death age"
+msgstr "Calcola l'età alla morte"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1394
+msgid "Whether to compute a person's age at death."
+msgstr "Indica se calcolare l'età di una persona alla morte oppure no."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1397
+msgid "Omit duplicate ancestors"
+msgstr "Omette gli antenati duplicati"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1398
+msgid "Whether to omit duplicate ancestors."
+msgstr "Indica se omettere gli antenati duplicati oppure no."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1401
+msgid "Use complete sentences"
+msgstr "Usa frasi complete"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1403
+msgid "Whether to use complete sentences or succinct language."
+msgstr "Indica se usa frasi complete oppure linguaggio succinto."
```

Comment:
The same sentence is "Indica se usare frasi complete oppure un linguaggio sintetico."